include/RAJA/config.hxx.in

```diff
@@ -154,102 +154,106 @@ const int COHERENCE_BLOCK_SIZE = @RAJA_COHERENCE_BLOCK_SIZE@;
 //
 //     RAJA_INLINE - macro to enforce method inlining
 //
-//     RAJA_ALIGN_DATA(<variable>) - macro to express alignment of data, 
-//                              loop bounds, etc.
+//     RAJA_SIMD - macro to apply compiler-specific pragma that encourages
+//                 compiler to SIMD vectorize loop even if it may not
+//                 normally
+//
+//     RAJA_SIMDFORCE - macro to apply compiler-specific pragma that forces
```

Comment:
NOTE: This isn't being used currently and I think it is somewhat unseemly (and potentially problematic. It's here for reference. Should we keep it.